Adyen/Components/AbstractPersonalInformationComponent/AbstractPersonalInformationComponent.swift

```diff
@@ -9,7 +9,7 @@ import UIKit
 
 /// An abstract class that needs to be subclassed to abstract away any component
 /// whoes form consists of a combination of personal information pieces like first name, last name, phone, email, and billing address.
-open class AbstractPersonalInformationComponent: PaymentComponent, PresentableComponent, Localizable {
+open class AbstractPersonalInformationComponent: PaymentComponent, PresentableComponent, Localizable, LoadingComponent {
```

Comment:
There is no `stopLoading` in this class

Reply:
its in an extension, moved the protocol conformance to the extension. 👍